### PR TITLE
`scripts/vm-util.py` additions

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

**Review by Dennisbonke — APPROVED** [review]
LGTM

File: scripts/vm-util.py [hunk 1]
```diff
@@ -277,6 +278,10 @@ def do_qemu(args):
         for device in args.usb_passthrough_pcap:
             qemu_args += qemu_process_usb_passthrough(device, True)
 
+    qemu_sudo = []
+    if args.sudo:
+        qemu_sudo = ["sudo"]
```
Comment (ArsenArsen) [inline scripts/vm-util.py:283]: launching graphical programs via sudo usually breaks, especially on wayland, and, more importantly, i'm not sure how good of an idea that is

Comment (KSPAtlas) [reply scripts/vm-util.py:283]: > launching graphical programs via sudo usually breaks, especially on wayland, and, more importantly, i'm not sure how good of an idea that is

Aren't there use cases for running QEMU with sudo, such as some networking configurations?

Comment (ArsenArsen) [reply scripts/vm-util.py:283]: I suspect it's never necessary - I'd need more details

**Comment by no92** [comment]
Dropped the commit adding `--sudo`. Instead, we check that the user has set up the passthrough device for rootless passthrough, and error out if not.

File: scripts/vm-util.py (hunk 1 above)
Comment (ArsenArsen) [reply scripts/vm-util.py]: I doubt it was unavoidable.  we'd probably benefit more from supporting libvirt than running graphical programs with root, have you tried that?

**Pull request merged**